package.json

```diff
@@ -129,7 +129,7 @@
     "@types/diff": "5.0.2",
     "@types/mime": "3.0.1",
     "@types/mocha": "10.0.1",
-    "@types/node": "18.14.6",
+    "@types/node": "20.2.5",
```

Comment:
Why is this needed? The export the change needs was added in Node 15.

Reply:
Not in the types package.